admin_solutions/master_node_config.adoc

```diff
@@ -0,0 +1,409 @@
+
+For Example:
+----
+networkConfig:
+  clusterNetworkCIDR: 10.3.0.0/16
+  hostSubnetLength: 8
+  networkPluginName: example/openshift-ovs-subnet
+# serviceNetworkCIDR must match kubernetesMasterConfig.servicesSubnet
+  serviceNetworkCIDR: 179.29.0.0/16
+----
+
+|`*volumeConfig*`
+a|Contains options for configuring volume plug-ins in the master node:
+
+- `*DynamicProvisioningEnabled*` (boolean): Default value is `true`, and toggles dynamic provisioning off when `false`.
+
+|===
+
+=== Node Configuration File Options
+
+[[node-config-options]]
+[cols="1,4"]
+.Node Configuration File Options
+|===
+|Option |Description
+
+|`*nodeName*`
+|The value of the `*nodeName*` (string) is used to identify this particular node in the cluster. If possible, this should be your fully qualified hostname. If you are describing a set of static nodes to the master, then this value must match one of the values in the list.
+
+|`*nodeIP*`
+|A node may have multiple IPs. This specifies the IP to use for pod traffic routing. If left unspecified, a network look-up is performed on the nodeName, and the first non-loopback address is used.
+
+|`*servingInfo*`
+|Describes how to start serving.
+
+|`*masterKubeConfig*`
+|The filename for the .kubeconfig file that describes how to connect this node to the master.
+
+|`*dnsDomain*`
+|Holds the domain suffix.
+
+|`*dnsIP*`
+|(string) Contains the IP. Can be controlled with openshift_dns_ip in the Ansible inventory.
+
+|`*networkPluginName,omitempty*`
+|Deprecated and maintained for backward compatibility, use `*NetworkConfig.NetworkPluginName*` instead.
+
+|`*networkConfig*`
+a|Provides network options for the node:
+
+- `*NetworkPluginName*` (string): Specifies the networking plug-in.
+- `*MTU*` (unsigned integer): Maximum transmission unit for the network packets.
+
+|`*volumeDirectory*`
+|The directory that volumes will be stored under.
+
+|`*imageConfig*`
+|Holds options that describe how to build image names for system components.
+
+|`*allowDisabledDocker*`
+|If this is set to `true`, then the Kubelet will ignore errors from Docker.  This means that a node can start on a machine that does not have Docker started.
+
+|`*podManifestConfig*`
+|Holds the configuration for enabling the Kubelet to create pods based from a manifest file or files placed locally on the node.
+
+|`*authConfig*`
+|Holds authn/authz configuration options.
+
+|`*dockerConfig*`
+|Holds Docker related configuration options.
+
+|`*kubeletArguments,omitempty*`
+|Key value pairs that are passed directly to the Kubelet that matches the Kubelet's command line arguments. These are not migrated or validated, so if you use them, then they may become invalid. Use caution, because these values override other settings in the NodeConfig which may cause invalid configurations.
```

Comment:
Is "Key value" correct? I'm thinking "Key/value" or "Key=value".

Also, does "NodeConfig" need markup, or changed to "node configuration"? Also, s/which/that